src/styles/parts/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  width: 200px;
+  height: 400px;
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  border: 1 solid #f3f3f3;
```

Comment:
There is a missing unit for the border width. It should be specified, for example, as '1px solid #f3f3f3'.